### DISCUSSION TITLE
Renovate or Dependabot?

### DISCUSSION THREAD

**Discussion by ovflowd** [nodejs/nodejs.org#5490] [Ideas]
Now that we have a streamlined branch and "no more feature branches" (such as `major/website-redesign`), we can adopt an automated solution for alerting us for Dependency Updates and automatically updating them.

There are two major contenders here:

- **Dependabot**, backed by GitHub
- **Renovate**, another great solution.

I'd like to hear the community's opinions and @nodejs/website team opinions on what we should adopt :)

**Comment by Trott** [comment]
We [use Dependabot in the core repo](https://github.com/nodejs/node/blob/eece8d755c69d393a7bcc14f9c9454020d91ed1d/.github/dependabot.yml).

**Comment by bmuenzenmeyer** [comment]
Dependabot integration with GitHub is really nice. But it was inferior in some ways, like grouping. I am excited for things like https://github.blog/changelog/2023-06-30-grouped-version-updates-for-dependabot-public-beta/

**Comment by ovflowd** [comment]
PR for Dependabot https://github.com/nodejs/nodejs.org/pull/5493
